src/main/groovy/edu/oregonstate/mist/osuevents/resources/FeedResource.groovy

```diff
@@ -335,7 +335,7 @@ class FeedEvent {
         bool ? "1" : null
     }
 
-    @JsonProperty("Contact name for event questions or disability accommodations")
```

Comment:
"Contact Name" is consistent with the field in the OSU Events API event object, and is the Localist custom field, so this change makes sense to me. I'm not sure what the logic was of changing this to the longer name - maybe a more detailed field name for users? If so, the custom field's name would need to be changed in Localist as well (https://support.localist.com/bulk-add/). "Contact Name" seems more concise/professional to me.

Reply:
Yes, in our localist staging, we were using the field "Contact name for event questions...etc". However in our localist production environment, it was just "Contact Name". Using the field name from staging was causing contact names not to import. I also updated our localist staging to use "Contact Name", this change will allow contact name to be imported via the feed in staging and production.